app/models/customer.rb

```diff
@@ -0,0 +1,15 @@
+class Customer < ApplicationRecord
+  validates :name, :phone, :address, :city, :state, :postal_code, presence: true
+
+  has_many :rentals
+
+  # scope :by_name, -> (name) { where(name: name) }
+
+  def movies_checked_out_count
+    return Rental.where(customer_id: self.id).where(returned: false).count
```

Comment:
This might be an instance where instead of `Rental.where(customer_id: self.id)` you could use `self.rentals`, if you find a way to filter on these with `returned == false` (maybe `.filter`?)

Reply:
Then again, I see the benefit in the style you wrote it in already here